.npmrc

```diff
@@ -0,0 +1,2 @@
+shamefully-hoist=true
```

Comment:
Nitro is compatible with pnpm hoist. And we shall not enable this flag to make sure it keeps compatible.